edgexfoundry/edgex-ui-go
# fix: AppService in the UI by adding a second auth field

closes #672 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [] I have added unit tests for the new feature or bug fix (if not, why?)

## Testing Instructions
* check out this branch
* issue `make docker` to create a new ui image
* enable security
* update `docker-compose.yml` with the local dev UI image
* bring up the docker environment
* open the UI see the new dialog
* from `edgex-compose` issue `make get-token` use this as the "Gateway Token" and `make get-consul-acl-token` for the "ACL Token"

## New Dependency Instructions (If applicable)
none
